### PR TITLE
Add Object Completion

### PR DESCRIPTION
**Pull request by WitherFlower**
idk what i'm doing but if i understood the code properly this should be pretty close

also i don't know how to test this thing so yeah

### CODE REVIEW

File: src/sql/queries.py [hunk 1]
```diff
@@ -1248,6 +1248,23 @@ def round_rng(rng):
         title = f"Daily Completion - {calendar.month_name[month]} {year}"
         range_arg = "DATE_PART('day', approved_date)"
         prefix = ""
+    elif type == "objects":
```
Comment (WitherFlower) [inline src/sql/queries.py:1251]: Note: I didn't really understand what the `di` thing was about so I didn't touch it, however I noticed in some cases we remove `-modded` from it so idk if this applies here

Comment (respektive) [reply src/sql/queries.py:1251]: Haven't looked at this in a while, but i believe the `-modded` key gets deleted for the queries where we wanna make sure it's using the nomod values, like for the stars or length. However its not being removed for the CS,OD or AR so idk

**Review by respektive — APPROVED** [review]
lgtm

**Comment by WitherFlower** [comment]
let's fucking go

**Pull request merged**